framework/scripts/find_hung_process.py

```diff
@@ -362,7 +363,7 @@ def __longest_trace(self):
         """
         return max(trace.count('\n') for trace in self.traces)
 
-    def process_traces(self, num_lines_to_keep):
+    def process_traces(self, num_lines_to_keep, significant_params):
```

Comment:
params_are_significant is a better name, let s keep that one

Reply:
what s up with that one?
```suggestion
    def process_traces(self, num_lines_to_keep, params_are_significant):
```
significant_params would be for the list of significant parameters